api/turing/server/api.go

```diff
@@ -69,13 +70,20 @@ func AddAPIRoutesHandler(r *mux.Router, path string, appCtx *api.AppContext, cfg
 
 func openapiValidationMiddleware(apiPath string, cfg *config.Config) (mux.MiddlewareFunc, error) {
 	if cfg.OpenapiConfig != nil && cfg.OpenapiConfig.ValidationEnabled {
+		// Choose between bundled openapi yaml file or development files
```

Comment:
Can we reduce the complexity here and instead just update `.env.development` file to point to development files, while for production builds (i.e. image distributions) use bundled spec file?

Reply:
Removed and reworked.